java/logs-in-context-log4j2/README.md

```diff
@@ -17,11 +17,13 @@ The result is JSON structured logs, with one JSON object per line, which have th
   "log.level": "INFO",
   "logger.name": "com.newrelic.app.Controller",
   "message": "A sample log message!",
-  "trace.id": "6aae93314fe034149cd85f07eac24bc5",
-  "span.id": "f1be31bc6e4471d8"
+  "trace_id": "6aae93314fe034149cd85f07eac24bc5",
+  "span_id": "f1be31bc6e4471d8"
 }
 ```
 
+The OpenTelemetry Log specification defines that when propagating [trace context in legacy formats](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/README.md#trace-context-in-legacy-formats), `trace_id` and `span_id` should be used. However, [New Relic structured logging conventions](https://github.com/newrelic/newrelic-exporter-specs/tree/master/logging) expect trace context to be propagated as `trace.id` and `span.id`. The [transform](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/transformprocessor) processor is defined in the collector config to replace `trace_id` => `trace.id`, and `span_id` => `span.id`. 
```

Comment:
Tyler already mentioned this, but would this be a good place to call out that they could just use the New Relic conventions for trace id and span id at the logger layout level, which may be more performant overall?